### ISSUE TITLE
add documentation + example for webchat support

### ISSUE THREAD

**Issue by f1-outsourcing** [conversejs/converse.js#3279]
I think it would be nice to have a clear and separate section added documenting 'website chat support'[1]. I am not really sure what the main use is for converse.js but I would guess it is in the area of online support. I was thinking of adding converse.js in some admin/user sign up area so users could ask for support during their registration procedure. 

However I have been messing trying to get a single window, the embedded window is partly out of view, huge, messing with reversed proxies, after first login I can't login any more etc. I am just wasting time to see if even this conversejs can even be used.

Please provide some section for a basic chat support via a web interface.

- overlay with just 1 simple single window, where you type your question, and someone answers it
- maybe overlay that stays available / keeps session when browsing the website
- files are only used/downloaded when user opens the chat
- no useless elements like listing groupchats, contacts, participants, buddies, contact requests etc
- instructions how to configure/setup your xmpp server this specific use
- why not offer these as demo pages, so people can see

Maybe just open here discussions for support? Your contact page[2] is like, "if your car is broke, we can fix it, but you have to come and drive it here first."

[1]
https://github.com/conversejs/converse.js
https://conversejs.org/docs/html/

[2]
https://conversejs.org/#contact

Your main page here on github is lacking

**Comment by jcbrand** [comment]
I'm going to close this as a `wontfix`.

My time and energy is limited and I've decided I'm not going to spend time on this. Although Converse can be used for a support chat, it's not my focus. As always, I will accept contributions if anyone else is interested in doing this work.

**Comment by f1-outsourcing** [comment]
Yes if I would contribute it would also require that it is minimalistic as not a 2MB download for just a simple chat.

**Comment by jcbrand** [comment]
Converse is much more than a simple support chat, and for the amount of features it has, around 2Mb (minified but uncompressed) for the JavaScript and CSS is reasonable. That said, I'm always looking for ways to prune unnecessary dependencies and have done so multiple times in the past, in order to get the bundle size to be as small as possible.

In any case, Converse is composable and made out of different plugins. So if one wanted to create a simple support chat, you could remove all the plugins that aren't relevant to that use-case, and thereby have a much smaller build.

**Comment by f1-outsourcing** [comment]
Oh that is good to know. It is always better to add/enhance some existing project than create something totally separate. When I have some time I will check this modular structure. So in theory, I could get it down to <200kb?

**Comment by jcbrand** [comment]
In theory yes, in practice the actual size depends on what's needed and included in the final build.
